src/dpq2/conv/time.d

```diff
@@ -68,28 +85,55 @@ if( is( T == TimeStampWithoutTZ ) )
     );
 }
 
+/++
+    Structure to represent PostgreSQL's Timestamp without time zone type.
+    Conversions between timestamp without time zone and timestamp with time zone normally assume that
+    the timestamp without time zone value should be taken or given as timezone local time.
+    A different time zone can be specified for the conversion using AT TIME ZONE.
++/
 struct TimeStampWithoutTZ
```

Comment:
Also I propose to rename this struct to `TimeStamp` because it represents both timestamps values.

(Also we can add `Typedefs` for `TimeStampWithTZ` and `TimeStampWithoutTZ` and return it from appropriate functions to avoid to confuse them in applications code. But I am isn't sure.)

Reply:
It will be a breaking change and can be done after this PR?